back/spec/acceptance/ideas_spec.rb

```diff
@@ -737,7 +737,8 @@
         end
 
         describe do
-          let(:idea_images_attributes) { [{ image: Base64.encode64(Rails.root.join("spec/fixtures/image#{rand(20)}.png").open.read) }] }
+          let(:idea_image) { file_as_base64 'header.jpg', 'image/jpeg' }
+          let(:idea_images_attributes) { [{ image: idea_image }] }
```

Comment:
The original test was failing with the new validation of image presence, even though creating an idea with an image was very definitely possible via the FE.

It was responding with a 422, and when I examined the json_response it was because of the 'Image can't be blank." error.

This helper seems to fix the issue